stdlib/2/pydoc.pyi

```diff
@@ -0,0 +1,176 @@
+    def docother(self, object: object, name: Optional[str] = ..., mod: Optional[str] = ..., parent: Optional[str] = ..., maxlen: Optional[int] = ..., doc: Optional[Any] = ..., *ignored) -> str: ...
+
+def pager(text: str) -> None: ...
+def getpager() -> Callable[[str], str]: ...
+def plain(text: str) -> str: ...
+def pipepager(text: str, cmd: str) -> str: ...
+def tempfilepager(text: str, cmd: str) -> str: ...
+def ttypager(text: str) -> str: ...
+def plainpager(text: str) -> str: ...
+def describe(thing: Any) -> str: ...
+def locate(path: str, forceload: bool = ...) -> object: ...
+
+text = ...  # type: TextDoc
+html = ...  # type: HTMLDoc
+
+class _OldStyleClass: ...
+
+def resolve(thing: Union[str, object], forceload: bool = ...) -> Optional[Tuple[object, str]]: ...
+def render_doc(thing: Union[str, object], title: str = ..., forceload: bool = ...) -> str: ...
+def doc(thing: Union[str, object], title: str = ..., forceload: bool = ...) -> str: ...
+def writedoc(thing: Union[str, object], forceload: bool = ...) -> None: ...
+def writedocs(dir: str, pkgpath: str = ..., done: Optional[Any] = ...) -> None: ...
+
+class Helper:
+    keywords = ...  # type: MutableMapping[str, Union[str, Tuple[str, str]]]
+    symbols = ...  # type: MutableMapping[str, str]
+    topics = ...  # type: MutableMapping[str, Union[str, Tuple[str, ...]]]
+    def __init__(self, input: Optional[IO[str]] = ..., output: Optional[IO[str]] = ...) -> None: ...
+    input = ...  # type: IO[str]
+    output = ...  # type: IO[str]
+    def __call__(self, request: Union[str, Helper, object] = ...) -> None: ...
+    def interact(self) -> None: ...
+    def getline(self, prompt: str) -> str: ...
+    def help(self, request: Union[str, Helper]) -> None: ...
+    def intro(self) -> None: ...
+    def list(self, items: List[Any], columns: int = ..., width: int = ...) -> None: ...
+    def listkeywords(self) -> None: ...
+    def listsymbols(self) -> None: ...
+    def listtopics(self) -> None: ...
+    def showtopic(self, topic, more_xrefs: str = ...) -> None: ...
+    def showsymbol(self, symbol) -> None: ...
+    def listmodules(self, key: str = ...) -> None: ...
+
+help = ...  # type: Helper
+
+# See Python issue #11182: "remove the unused and undocumented pydoc.Scanner class"
+# class Scanner:
+#     roots = ...  # type: Any
+#     state = ...  # type: Any
+#     children = ...  # type: Any
+#     descendp = ...  # type: Any
+#     def __init__(self, roots, children, descendp) -> None: ...
+#     def next(self): ...
+
+class ModuleScanner:
+    quit = ...  # type: bool
+    def run(self, callback: Callable, key: Optional[Any] = ..., completer: Optional[Callable] = ..., onerror: Optional[Callable] = ...) -> None: ...
+
+def apropos(key: str) -> None: ...
+def serve(port: int, callback: Optional[Callable] = ..., completer: Optional[Callable] = ...) -> None: ...
```

Comment:
completer here is also a `Callback[[], None]`. `callback` takes a single argument.